docs/update_and_migration/from_4.4/update_from_4.4.md

```diff
@@ -236,31 +236,32 @@ Run the following scripts:
 
 === "MySQL"
 
-    ``` sql
+    ``` bash
     mysql -u <username> -p <password> <database_name> < vendor/ibexa/installer/upgrade/db/mysql/ibexa-4.5.1-to-4.5.2.sql
     ```
 
 === "PostgreSQL"
 
-    ``` sql
+    ``` bash
     psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-4.5.1-to-4.5.2.sql
     ```
 
 ### v4.5.3
 
-#### Database update
+#### Database update [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
 
+Update the database if you are using [[= product_name_exp =]] or [[= product_name_com =]]. [[= product_name_content =]] does not require the database update.
 Run the following scripts:
```

Comment:
```suggestion
If you are using [[= product_name_exp =]] or [[= product_name_com =]], run the following scripts:
```
or actually

```suggestion
Run the following scripts:
```
I don't think you need the sentence at all. We've been using badges above and I think their meaning is clear - whatever is marked with one applies to the indicated edition.

Thus my suggestion, which applies to the other occurrence as well.